app/navigation/VariationsNavigator.scala

```diff
@@ -37,7 +37,7 @@ class VariationsNavigator @Inject()(config: FrontendAppConfig) extends Navigator
       case Some(DeclareTheTrustIsUpToDate) =>
         controllers.routes.DeclarationController.onPageLoad(draftId)
```

Comment:
This is the incorrect DeclarationController in the navigator.
Is there a unit test for this?

Reply:
Amended to redirect to correct controller.
Unit tests for VariationsNavigator updated.